jonathaneeckhout/jdungeon3d
# New: add mushroomdog and wander behavior

Adds:
- Enemy class
- Mushroomdog scene
- Wander behavior
- AI navigation

How to test:
Run the game, run towards the coastline and check the mushroomdogs running around.
